virttest/cartesian_config.py

```diff
@@ -49,7 +49,7 @@
 
 ::
 
-    E = {\\n, #, :, "-", =, +=, <=, ?=, ?+=, ?<=, !, < , del, @, variants, include, only, no, name, value}
+    E = {\\n, #, :, "-", =, ~=, +=, <=, ?=, ?+=, ?<=, !, < , del, @, variants, include, only, no, name, value}
 
```

Comment:
I would recommend keeping the above suggested order here too, i.e. place the ``~=`` operator after ``?<=``.